website/src/components/Buttons/LikertButtons.tsx

```diff
@@ -0,0 +1,45 @@
+import { Button, SimpleGrid } from "@chakra-ui/react";
+import { PropsWithChildren, ReactNode } from "react";
+
+export const LikertButtons = ({
+  isDisabled,
+  options,
+  value,
+  onChange,
+}: PropsWithChildren<{
+  isDisabled: boolean;
+  options: ReactNode[];
+  value: number;
+  onChange: (number) => void;
+}>) => {
+  return (
+    <SimpleGrid
+      aria-roledescription="radiogroup"
+      data-cy-complete={value !== null}
+      columns={options.length}
+      spacing={[1, 4]}
+    >
+      {options.map((option, idx) => {
+        const indexValue = idx / (options.length - 1);
```

Comment:
I am not really sure I understand what the purpose of this `indexValue` is

Reply:
The likert buttons are a series of N buttons, the values passed in are normalised 0.0-1.0, the indexValue is the value from 0.0-1.0 represented by button i of N, the value that the button at this index represents/controls.